### DISCUSSION TITLE
Use Policy Value on TPPObject

### DISCUSSION THREAD

**Discussion by TimNFL** [Venafi/venafips#118] [Q&A] [answered]
I've been trying to figure out the correct way to clear an attribute from a TPPObject so that the value reverts to the Policy Value. I have tried syntax like this

`Set-TppAttribute -Path '\VED\Policy\My Folder\app.company.com' -Attribute @{'My custom field Label'=$null}`
which errors with
`Invoke-WebRequest : {"Error":"Internal error occurred."}`

and the value is set as an empty string

Then I tried something like this:
`Set-TppAttribute -Path '\VED\Policy\My Folder\app.company.com' -Attribute @{'My custom field Label'=''}`

but that just sets the value to an empty string.

after reading the WebSDK documentation, thought "null" might me a special keyword so I tried something like this

`Set-TppAttribute -Path '\VED\Policy\My Folder\app.company.com' -Attribute @{'My custom field Label'='null'}`

But that literally set the value to the word null. I know I am overlooking something. Can someone please help me understand how to clear a value so it reverts back to the policy value.

Thanks

TimFromFL

**Comment by gdbarron** [comment]
This is a bug with `Set-TppAttribute`.  #119 opened.

**Reply by gdbarron** [reply, marked as answer]
Fixed in #120
